### PR TITLE
Humble migration

### PR DESCRIPTION
**Pull request by arekin19**
We are porting our internal modifications and features from galactic to humble.

### CODE REVIEW

File: nav2_bt_navigator/src/bt_navigator.cpp [hunk 1]
```diff
@@ -115,6 +115,7 @@ BtNavigator::on_configure(const rclcpp_lifecycle::State & /*state*/)
 
   pose_navigator_ = std::make_unique<nav2_bt_navigator::NavigateToPoseNavigator>();
   poses_navigator_ = std::make_unique<nav2_bt_navigator::NavigateThroughPosesNavigator>();
+  nodes_navigator_ = std::make_unique<nav2_bt_navigator::NavigateFleetNodesNavigator>();
```
Comment (TanmayDeshmukh) [inline nav2_bt_navigator/src/bt_navigator.cpp]: Oh actually @cmraaron we're not using this anymore right? 
Would be nice to minimize our custom changes to nav2 so we don't carry it forward

File: nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp [hunk 2]
```diff
@@ -142,6 +145,14 @@ void RegulatedPurePursuitController::configure(
       logger_, "approach_velocity_scaling_dist is larger than forward costmap extent, "
       "leading to permanent slowdown");
   }
+  node->get_parameter(
```
Comment (TanmayDeshmukh) [inline nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp]: oh no these lines are repeated in RPP.. We might need to drop some of our pervious cherry-picked commits

**Comment by dimaxano** [comment]
Why that PR is in draft?

File: nav2_velocity_smoother/src/velocity_smoother.cpp [hunk 3]
```diff
@@ -115,7 +115,7 @@ VelocitySmoother::on_configure(const rclcpp_lifecycle::State &)
   // Setup inputs / outputs
   smoothed_cmd_pub_ = create_publisher<geometry_msgs::msg::Twist>("cmd_vel_smoothed", 1);
   cmd_sub_ = create_subscription<geometry_msgs::msg::Twist>(
-    "cmd_vel", rclcpp::QoS(1),
+    "cmd_vel_input", rclcpp::QoS(1),
```
Comment (arekin19) [inline nav2_velocity_smoother/src/velocity_smoother.cpp]: We are dropping this commit. Remapping will be done fully through serena_bringup

File: nav2_bt_navigator/src/bt_navigator.cpp (hunk 1 above)
Comment (cmraaron) [reply nav2_bt_navigator/src/bt_navigator.cpp]: indeed, we are not

**Review by cmraaron — DISMISSED** [review]

**Review by dimaxano — DISMISSED** [review]

**Review by dimaxano — DISMISSED** [review]

File: nav2_velocity_smoother/src/velocity_smoother.cpp (hunk 3 above)
Comment (TanmayDeshmukh) [reply nav2_velocity_smoother/src/velocity_smoother.cpp]: Done with this commit in serena_bringup: https://github.com/cmrobotics/serena_bringup/pull/136/commits/161695921520ffc50b4df6ca716eea18f397a3f4

**Comment by TanmayDeshmukh** [comment]
- dropping commits that have custom changes to bt_navigator (SS-185) as navigate_fleet_nodes is not used anymore.
- dropping custom changes to velocity smoother that changes topic name and using remapping in bringup launch instead (SS-186).

**Review by TanmayDeshmukh — APPROVED** [review]

**Review by dimaxano — APPROVED** [review]

**Pull request merged**